### ISSUE TITLE
Mask R-CNN configs

### ISSUE THREAD

**Issue by xwjabc** [whai362/PVT#18]
Hi, thank you for your great work! Recently we would like to compare your model with ours on the Mask R-CNN results. I wonder if you can provide some configs for Mask R-CNN settings? Thanks!

**Comment by whai362** [comment]
Thanks for your attention!
I will push the config of Mask R-CNN in a week.

**Comment by xwjabc** [comment]
By the way, I attempted to use PVT as a backbone in Detectron2 with the ImageNet pretrained model. However, under the Mask R-CNN w/ FPN 1x setting, I found the performance of PVT-Small is even worse than the ResNet-50 model. Since the original PVT paper does not mention the results under this setting, I wonder if this phenomenon is normal, or 3x is required to make PVT work. Thanks!

**Comment by whai362** [comment]
What is your optimizer and init learning rate?
In RetinaNet, we change:
(1) Backbone;
(2) Optimizer: Adam;
(3) LR: 1/100 of the one when using SGD.

**Comment by xwjabc** [comment]
> What is your optimizer and init learning rate?
> In RetinaNet, we change:
> (1) Backbone;
> (2) Optimizer: Adam;
> (3) LR: 1/100 of the one when using SGD.

Thank you for your reply! I use the default Mask R-CNN setting in Detectron2: 
- Optimizer: SGD with Momentum
- Learning rate: 0.02 for batch size 16
I wonder if you also change the optimizer and learning rate settings in Mask R-CNN settings as well as the ones in RetinaNet settings? Thanks!

**Comment by xwjabc** [comment]
Besides, in the PVT paper, I found it said:
Our models are trained with the batch size of 16 on 8 V100 GPUs and optimized by **AdamW** with the initial learning rate of **1×10^-4**.
Besides, in `detection/configs/retinanet_pvt_s_fpn_1x_coco.py`, it has the optimizer settings:
```
optimizer = dict(type='AdamW', lr=0.0001, weight_decay=0.0001)
optimizer_config = dict(grad_clip=None)
# learning policy
lr_config = dict(
    policy='step',
    warmup='linear',
    warmup_iters=500,
    warmup_ratio=0.001,
    step=[8, 11])
```
Thus, I wonder if AdamW with 1e-4 is used in Mask R-CNN as well (seems in the paper the RetinaNet and Mask R-CNN settings are mentioned together), since 1e-4 is 1/200 of the 0.02 which is the original learning rate in Detectron2.

**Comment by whai362** [comment]
It is a clerical error. In Mask R-CNN, the LR is 2e-4.
We use Adam because it is difficult for SGD to optimize the transformer.

**Comment by xwjabc** [comment]
> It is a clerical error. In Mask R-CNN, the LR is 2e-4.
> We use Adam because it is difficult for SGD to optimize the transformer.

Got it. I will try the optimization setting you provided for another experiment. Thanks!

**Comment by feiyuhuahuo** [comment]
@xwjabc , Hi, have you got any result after using optimization setting. I also met the same problem as you did.

**Comment by xwjabc** [comment]
> @xwjabc , Hi, have you got any result after using optimization setting. I also met the same problem as you did.

Not yet. Will start some experiments these days.

**Comment by xwjabc** [comment]
Hi @whai362, I saw in detection folder in this repo, the Mask R-CNN 1x config uses FPN, but the paper seems to say that this config does not use FPN. I wonder if FPN is used or not?
